src/components/MoviePage.jsx

```diff
@@ -1,23 +1,34 @@
-import React from 'react';
+import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import Results from './Results';
 import Content from './Content';
 import Footer from './Footer';
 import MovieDetails from './MovieDetails';
+import { connect } from 'react-redux';
 
-const SearchPage = ({ movies }) => {
-  return (
-    <div>
-        <MovieDetails movie={movies[0]} />
+class MoviePage extends Component {
+  render() {
+    const { movies } = this.props;
+    return (
+      <div>
+        <MovieDetails />
         <Results items={movies.length} />
         <Content movies={movies} />
```

Comment:
`movies` prop is already set inside `Content.jsx` file (see `mapStateToProps`)

Reply:
Fixed.